lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/client/LocalRuntimeConfiguration.java

```diff
@@ -80,6 +83,19 @@
    * the Driver (and the Evaluators) also use it.
    */
   public static final OptionalImpl<TcpPortProvider> TCP_PORT_PROVIDER = new OptionalImpl<>();
+  /**
```

Comment:
Are these needed for the client? I though the issue only exists on the Driver / the Evaluators?